nservicebus/tools/migrate-to-native-delivery.md

```diff
@@ -41,15 +41,15 @@ The current version of the tool supports the following persisters:
 - [RavenDB persistence](/persistence/ravendb) versions 3.5.x and 4.x of the RavenDB database server
 - [Azure Storage persistence](/persistence/azure-table) version 2.4.x compatible timeouts stored in Azure Storage tables
 
-NOTE: Additional targets will be added before NServiceBus version 8 is released. Namely Amazon SQS transport as a target.
-
 ## Supported transports
 
 The tool supports the following transports:
 
-- [RabbitMQ](/transports/rabbitmq/)
-- [SQL Transport](/transports/sql)
-- [Azure Storage Queues](/transports/azure-storage-queues/)
+- [RabbitMQ transport](/transports/rabbitmq/)
+- [SQL transport](/transports/sql)
+- [Azure Storage Queues transport](/transports/azure-storage-queues/)
+
+NOTE: Support for the [Amazon SQS transport](/transports/sqs/) will be added before NServiceBus version 8 is released.
```

Comment:
```suggestion
```

Reply:
This can be removed completely as we don't need to support Amazon SQS in the tool.